### DISCUSSION TITLE
Proposal:link to a label in another typst file

### DISCUSSION THREAD

**Discussion by Feiar** [typst/typst#4529] [General]
For example, in a web page
```A.html: <a id="anchor">anchor</a>```
```B.html: <a href="A.html#anchor">jump to anchor</a>```
Can #link function adds similar features ?

**Comment by matthiasbeyer** [comment]
Just came across this because I was researching how this would work. I would like to see this feature as well.

**Reply by hpcfzl** [reply]
This is already merged, see the explanation in https://github.com/typst/typst/pull/7964.
